packages/idyll-cli/src/pipeline/index.js

```diff
@@ -1,7 +1,7 @@
 const fs = require('fs');
-const Promise = require('bluebird');
```

Comment:
Great! It's nice to get rid of these old dependencies now that they're widely available in node.